brayovsky/pivotaly
# Ft edit project id 159686635

#### What does this PR do?
Adds a control panel that:
##### Features
- Allows a user to edit his/her token
- Allows a user to edit the project id set
- Allows dynamic resetting of token or project id when a wrong one is detected
##### Technical debt
- Adds validator tests to increase coverage

#### How should this be manually tested?
Start the app and click on the pivotaly logo. A control panel view should appear that allows you to edit your token and project

#### Any background context you want to provide?
It was only possible to edit these before after restarting visual studio code.

#### Screenshots (if appropriate)
<img width="369" alt="Screen Shot 2019-05-10 at 20 57 32" src="https://user-images.githubusercontent.com/19430730/57546951-4ef44300-7366-11e9-85c5-522b0db2336c.png">


#### Questions:
n/a
